CHANGELOG.md

```diff
@@ -1,4 +1,66 @@
 # Changelog
+
+## [v1.12.0](https://github.com/mongodb/terraform-provider-mongodbatlas/tree/v1.12.0) (September 19, 2023)
+
+[Full Changelog](https://github.com/mongodb/terraform-provider-mongodbatlas/compare/v1.11.1...v1.12.0)
+
+**Enhancements**
+
+- Support new tags attribute in cluster, advanced\_cluster, and serverless\_instance [\#1461](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1461) ([AgustinBettati](https://github.com/AgustinBettati))
+- Add database\_user username parameter OIDC footnote in docs [\#1458](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1458) ([Zuhairahmed](https://github.com/Zuhairahmed))
+- Support for new mongodbatlas\_atlas\_user and mongodbatlas\_atlas\_users data sources [\#1432](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1432) ([AgustinBettati](https://github.com/AgustinBettati))
+
+**Bug Fixes**
+
+- Populate total\_count in mongodbatlas\_alert\_configurations data source  [\#1476](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1476) ([lantoli](https://github.com/lantoli))
+- Improved error handling for cloud\_backup\_schedule resource. [\#1474](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1474) ([marcosuma](https://github.com/marcosuma))
+- Handle incorrect ids when importing alert\_configuration or project\_ip\_access\_list resources [\#1472](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1472) ([lantoli](https://github.com/lantoli))
+- Fix when removing collectionName from user role [\#1471](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1471) ([marcosuma](https://github.com/marcosuma))
+- Changelog Spelling Fixes  [\#1457](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1457) ([Zuhairahmed](https://github.com/Zuhairahmed))
+
+**Deprecations and Removals**
+
+- Deprecation of labels attributes in cluster and advance_cluster resources.
+- Remove 1.12.0 deprecated fields. [\#1418](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1418) ([marcosuma](https://github.com/marcosuma))
+- Remove deprecated fields in alert\_configuration resource [\#1385](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1385) ([AgustinBettati](https://github.com/AgustinBettati))
+- Remove api\_keys attribute from mongodbatlas\_project [\#1365](https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1365) ([maastha](https://github.com/maastha))
```

Comment:
Another removal was in encryption_at_rest (removed deprecated fields `aws_kms`, `azure_key_vault`, `google_cloud_kms`)
My bad, should have been a separate PR.

PR https://github.com/mongodb/terraform-provider-mongodbatlas/pull/1383/files#diff-72229c36539e51cfa6fbd6c54b91a9b3ab72a524afce9428734c0c222c5f4528

Reply:
thx, added in new commit